.travis.yml

```
@@ -84,6 +84,15 @@ script:
     if [[ -v MAVEN_CHECKS ]]; then
       ./mvnw install -DskipTests -B -T C1 -P ci
     fi
+  - |
+    if [[ -v BUILD_PRESTO_DOCKER ]]; then
+      PRESTO_VERSION=$(./mvnw --quiet --batch-mode --non-recursive  exec:exec -Dexec.executable='echo' -Dexec.args='${project.version}')
```

Since this became longer now, it needs:

```suggestion
      set -xeu
      PRESTO_VERSION=$(./mvnw --quiet --batch-mode --non-recursive  exec:exec -Dexec.executable='echo' -Dexec.args='${project.version}')
```

---

This has been moved to `presto-docker/build.sh`